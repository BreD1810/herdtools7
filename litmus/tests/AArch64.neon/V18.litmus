AArch64 V18
(* Tests MOV from general *)
{
x=1;
0:X0=x;0:X1=y;0:X2=z;
}

 P0                     ;
 LDR W3,[X0]            ;
 MOV V0.B[0],W3         ;
 ST1 {V0.B}[0], [X1]    ;
 ST1 {V0.B}[1], [X2]    ;
 forall (y=1 /\ z=0)
