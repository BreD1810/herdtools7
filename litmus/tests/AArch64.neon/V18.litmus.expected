Test V18 Required
Histogram (1 states)
1000000:>y=1; z=0;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=1 /\ z=0) is validated
Hash=ad41edd5b0e91b7146fa32c45dc8c6c3
Observation V18 Always 1000000 0
